plugins/woocommerce/changelog/47680-fix-only-sticky-summary-if-smaller-than-screen

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+Comment: This is a follow up to another PR not yet shipped.
```

Comment:
Hi Nadir, quick question, can you link the other PR please? Can this be reviewed and merged first?

Reply:
The other PR is already merged

Reply:
https://github.com/woocommerce/woocommerce/pull/47376